terraform/modules/standards/outputs.tf

```diff
@@ -10,12 +10,6 @@ output "service" {
   value       = local.service
 }
 
-output "region_name" {
-  description = "**Deprecated**. Use `primary_region.name`. The region name associated with the current caller identity"
-  sensitive   = false
-  value       = data.aws_region.this.name
-}
-
```

Comment:
As noted, this was already deprecated in favor of the `primary_region` output. With updates to aws provider version 6, `data.aws_region` has a deprecated `name` and an attribute, in favor of `region`, so... references like this would be most correct as `data.aws_region.this.region`. ¯\_(ツ)_/¯